### PR TITLE
README.md: expand a bit on how we use this client

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

To help distinguish between this and sul_orcid_client

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, including data writes to shared file systems, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Test Coverage](https://api.codeclimate.com/v1/badges/25b4a4111f831121dda5/test_coverage)](https://codeclimate.com/github/sul-dlss/mais_orcid_client/test_coverage)
 
 # mais_orcid_client
-API client for accessing MAIS's ORCID endpoints.
+API client for accessing MAIS's ORCID endpoints, e.g. using MAIS to check ORCID permissions and getting the access token for ORCID.
```
Comment (justinlittman) [inline README.md:7]: This is only some of the methods supported by this gem.

**Review by justinlittman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ndushay) [reply README.md:7]: I would be happy to add an "or" clause, or change this to say something like "to push/pull from orcid, use this client for XXX and use sul_orcid_client for xxx".

Comment (justinlittman) [reply README.md:7]: How about something like:
> MAIS's ORCID API provides access to ORCID information for Stanford users. (This is different from orcid.org's ORCID API, which is supported by https://github.com/sul-dlss/orcid_client.)

Comment (ndushay) [reply README.md]: took your suggestion.

**Review by justinlittman — APPROVED** [review]

**Pull request merged**